packages/backend-core/src/users/db.ts

```diff
@@ -44,6 +44,12 @@ type GroupFns = {
   getBulk: GroupGetFn
   getGroupBuilderAppIds: GroupBuildersFn
 }
+type CreateAdminUserOpts = {
+  ssoId?: string
+  hashPassword?: boolean
+  requirePassword?: boolean
+  noPasswordValidation?: boolean
```

Comment:
nit: it would be a lot nicer if this was a truthy name, e.g. `validatePassword`. The usage later on, `if (!opts.noPasswordValidation)` makes me pause for a sec when looking at it.

Reply:
I had a look at this - I fear that might require a lot more validation, as any call that sets options would then need to set the `validatePassword` - or we would need to do an `undefined` check to default it to true - setting it disable the password validation is a bit more confusing but feels a bit safer as the default is always validate.

Reply:
That could be called `skipPasswordValidation` and it sounds less negative :)